### PR TITLE
Ethereum test coverage report

### PR DESCRIPTION
**Pull request by rgarc**
This PR does a couple of things
- Adds a new CI workflow for running the ethereum test suite at midnight, Monday - Friday
- Automatically sends an email containing a stylized test coverage report

### CODE REVIEW

File: scripts/ethereum-test/send-test-coverage-report.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+SUMMARY_FILENAME=summary_${DATE}.html
+COMPLETE_RESULTS_FILENAME=complete_results_${DATE}.html
+
+ARGS='--repo_url '${REPO_URL}' --commit '${CURRENT_COMMIT}' --date '${DATE}
+
+python3 ./scripts/ethereum-tests/render_results.py $ARGS --file results.json --summary >${SUMMARY_FILENAME}
+python3 ./scripts/ethereum-tests/render_results.py $ARGS --file results.json >${COMPLETE_RESULTS_FILENAME}
+python3 ./scripts/ethereum-tests/send_mail.py ${ARGS} \
+	--gmail_user ${GMAIL_APP_USER} \
+	--gmail_password ${GMAIL_APP_PASSWORD} \
```
Comment (sheapwilliams) [inline scripts/ethereum-test/send-test-coverage-report.sh:16]: Whose creds are here?  Hopefully some sort of service account?

File: scripts/ethereum-test/send_mail.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    summary_filename = args.summary
+    complete_results_filename = args.complete_results
+    date = args.date
+
+    port = 465
+    smtp_server = "smtp.gmail.com"
+    sender_email = gmail_user
+    recipients = [
+        # TODO: add more recipients
+        "rene@monad.xyz",
```
Comment (sheapwilliams) [inline scripts/ethereum-test/send_mail.py]: Probably should just go to a mailing list or something easier to join/unjoin?

File: scripts/ethereum-test/send-test-coverage-report.sh (hunk 1 above)
Comment (rgarc) [reply scripts/ethereum-test/send-test-coverage-report.sh:16]: Right now, nobody. This PR needs @jhunsaker to add the respective secrets to the repository
https://github.com/monad-crypto/monad/blob/bd014c8035a544d233e7d4a3c80798779b083a8c/.github/workflows/ethereum-tests.yaml#L54-L55

File: scripts/ethereum-test/send_mail.py (hunk 2 above)
Comment (rgarc) [reply scripts/ethereum-test/send_mail.py]: Took me an embarrassing amount of time to figure out how to create a mailing list 😂 
https://groups.google.com/a/monad.xyz/g/monad-ethereum-tests

**Pull request closed without merging**